Add a trademark policy (#57)
NixOS/foundation: trademark/policy.md

Adapted from the [python trademark usage policy](https://www.python.org/psf/trademarks/)

# Introduction

This document outlines the policy of the NixOS Foundation (“NF”) regarding the use of its trademarks. Any use of any NF trademark must be in accordance with this policy.

> “Nix” and the Nix logo is a registered trademark of the PSF.

Author: The python policy also specifies some non-registered trademarks here (“use trademarks”). We can do the same if it makes sense

> “Nix” and the Nix logo is a registered trademark of the PSF.

Reviewer: ```suggestion
“Nix” and the Nix logo is a registered trademark of the NF.
```

1. If the trademark is registered with the U.S. Patent and Trademark Office, it is referred to as a registered mark. The first or most prominent mention of a Nix trademark should be immediately followed by a symbol for registered trademark: “®” or “(r)”. For example “Nix® …” This requirement is waived in all contexts where such marks are not normally included: email, online discussion, non-graphical advertisements (when permitted), and academic papers. We encourage the use of the symbol whenever possible, but recognize that many non-commercial and informal uses will omit it.
2. Try to give the word “Nix” distinctive graphic treatment wherever possible. The trademark should be set apart from surrounding text by using ALLCAPS, *italics*, **emphasized** or underlined fonts.
3. If the word “Nix” or the Nix logos are used in certain contexts, the following statement should accompany its use: “*Nix” and the Nix logos are trademarks or registered trademarks of the Nix Software Foundation, used by ___________ with permission from the Foundation.*
    

>     For websites and documentation this can be on a “legal statements” page. For brochures and published articles, this statement is optional. We encourage use of this statement, particularly for published materials, but recognize some non-commercial and informal uses will omit it.

Author: I'm not sure whether we want that. Feels very heavy and corporate

    
4. Always use any trademark as an adjective only, followed by a generic noun. For instance, it is correct to refer to the Nix programming language (adjective) but not simply to Nix (noun). Don't use the trademark as a verb (“Nix your software today!”).

## Examples

We have specific rules for the following uses:

1. Use of the word “Nix” in text, or as text in 3rd party logos and trademarks.
2. Use of one of the NF-provided logo variants in *unaltered* form.

> 3. Use of a logo *derived* from the Nix logos. For example, use of the intertwined snake graphic combined with different text, or with no text, or in combination with other graphic elements.

Author: ```suggestion
3. Use of a logo *derived* from the Nix logos. For example, use of the Nix snowflake graphic combined with different text, or with no text, or in combination with other graphic elements.
```

- Use of the word “Nix” in company names – Allowed only by prior written permission from the NF.
- Use of the word “Nix” when redistributing the Nix programming language as part of a freely distributed application – Allowed. If the standard version of the Nix programming language is modified, this should be clearly indicated. For commercial distributions, [contact the NF](mailto:foundation@nixos.org) for permission if your use is not covered by the nominative use rules described in the section “[Uses that Never Require Approval](#uses-that-never-require-approval)” above.
- Use of the word “Nix” in the names of user groups and conferences that are free to join or attend (Ex., “Dallas Nix Users Group”) – Allowed if for the Nix programming language. Other uses require permission.
- Use of the word “Nix” in the name of books or publications like “Nix Journal” and “Nix Cookbook” – Allowed if for the Nix package manager.
- Use of the word “Nix” on websites, brochures, documentation, and product packaging – Allowed if referring to the Nix package manager. Please follow the rules above about the use of the circle-R symbol.

> - Use of the word “Nix” in advertisements – Allowed in most cases by the nominative use rules described in the section “[Uses that Never Require Approval](https://www.python.org/psf/trademarks/#uses-that-never-require-approval)” above. Other uses in ads only with prior permission.

Author: ```suggestion
- Use of the word “Nix” in advertisements – Allowed in most cases by the nominative use rules described in the section “[Uses that Never Require Approval](#uses-that-never-require-approval)” above. Other uses in ads only with prior permission.
```

- Use of the word “Nix” in email and informally – Allowed without the circle-R symbol.
- Use of the word “Nix” in academic papers, theses, and books – Allowed without the circle-R symbol. Books should include the symbol.
- Use of the word “Nix” in another trademark – Not allowed without prior written permission from the NF, except as described above.

### Unaltered Logos

- Use of unaltered NF-provided logos on T-shirts, mugs, etc. – Again, non-commercial uses to promote the Nix programming language are allowed. Commercial uses (which includes any use where you sell these items for money) require permission from NF. Please reproduce our logos with the right colours and fonts; if you need help, let us know.

> - Use of unaltered NF-provided logos on websites, brochures, and product packaging. The “snowflake” graphic alone is an unaltered version, whether or not accompanied by the words in NF-provided logos. Non-commercial uses to promote the Nix package manager are allowed, as are all nominative uses as described in the section “[Uses that Never Require Approval](https://www.python.org/psf/trademarks/#uses-that-never-require-approval)”. Any other commercial uses require prior written permission from NF.

Author: ```suggestion
- Use of unaltered NF-provided logos on websites, brochures, and product packaging. The “snowflake” graphic alone is an unaltered version, whether or not accompanied by the words in NF-provided logos. Non-commercial uses to promote the Nix package manager are allowed, as are all nominative uses as described in the section “[Uses that Never Require Approval](#uses-that-never-require-approval)”. Any other commercial uses require prior written permission from NF.
```

- Use of freely distributable *derived* logos as icons for files and executables – Allowed if used to refer to the Nix package manager. Commercial users should obtain permission before using *derived* logos as icons for proprietary file formats.
- Use of *derived* logos for user groups and conferences – Allowed if used to refer to the Nix package manager. Commercial user groups and for-profit conferences require permission from the NF.
- Use of *derived* logos for freely distributed 3rd-party modules or tools – Allowed if for the Nix package manager. Use of *derived* logos for commercial modules and tools requires permission from the NF.
- We recommend [contacting the NF](mailto:foundation@nixos.org) for permission for **all** *derived* logos to avoid placing a confusing logo into wide-spread use. Contacting us is not a requirement for the specific non-commercial uses listed above, or when using freely distributable *derived* logos that have already been approved by the NF. However, obtaining permission from the NF is required in all other uses of a *derived* logo.

> 

Author: ```suggestion

# Notes

```

Underlying NF's trademark policy is a set of guidelines for what is – and is not – acceptable use of NF's trademarks, specifically the word mark “Nix”, the Python logos, and variations of those marks. This policy describes the uses generally approved by NF for its trademarks. However, if you violate this policy, or otherwise take actions that may compromise the goodwill or trademarks of NF, or expose NF to liability, NF may require you to cease all use of any NF trademark, regardless of the uses allowed in this policy.

# General Goals

> In general, we want the word mark “Nix” and the Nix logos to be used with minimal restriction to refer to the Nix programming language.

Reviewer 2: Programming language probably doesn't really apply. That's one of the less interesting parts of Nix to trademark. OS and package manager are the important ones IMO.

As such, stating accurately that software is written in the Nix programming language, that it is compatible with the Nix programming language, or that it contains the Nix programming language, is always allowed. In those cases, you may use the word “Nix” or [the unaltered logos](https://github.com/NixOS/nixos-artwork/blob/master/logo/README.md) to indicate this, without our prior approval. This is true both for non-commercial and commercial uses.

This clause overrides other clauses of this policy. However, if you have any doubts about your intended use of the trademarks, please [contact the NF board.](https://www.notion.so/Role-of-the-NixOS-Foundation-1640a3aa20c64f2aac9f15de34516150)

# Uses that Always Require Approval

Any commercial use of the NF trademarks in product or company names must be approved first by the NF. Some uses, like calling a company “The Nix Company,” or a product “Nix Language” or “Nix IDE” will be refused. This is because they are overly broad, or confusing whether the Nix programming language is open source or commercial, or whether your product or organization is affiliated with or sponsored by NF.

> Any use of a derived (modified) logo for any commercial purpose must also be approved first by the NF. We will generally be unable to do this, because of the confusion it may cause. As a guideline, modifications that leave the shape – but not necessarily the colours – unaltered are likely to be approved. Inclusion of other visual elements at an offset to the logo is generally acceptable (subject to other terms of this policy). Modifications that modify or obscure any part of the shape of the logo will not be approved.

Reviewer 2: I think this is overly strict. I've seen great derivatives of the Nix logo that would violate this policy but are obviously not official.

The most critical is probably the use of a mostly unaltered logo such as the NixOS release logos, the Nix community one or just generally text with the NixOS logo beside it.

- Use of the word “Nix” in academic papers, theses, and books – Allowed without the circle-R symbol. Books should include the symbol.
- Use of the word “Nix” in another trademark – Not allowed without prior written permission from the NF, except as described above.

### Unaltered Logos

- Use of unaltered NF-provided logos on T-shirts, mugs, etc. – Again, non-commercial uses to promote the Nix programming language are allowed. Commercial uses (which includes any use where you sell these items for money) require permission from NF. Please reproduce our logos with the right colours and fonts; if you need help, let us know.
- Use of unaltered NF-provided logos on websites, brochures, and product packaging. The “snowflake” graphic alone is an unaltered version, whether or not accompanied by the words in NF-provided logos. Non-commercial uses to promote the Nix package manager are allowed, as are all nominative uses as described in the section “[Uses that Never Require Approval](#uses-that-never-require-approval)”. Any other commercial uses require prior written permission from NF.

### Derived Logos

> - *Derived* logos must always be sufficiently different from the Nix logos to allow the community to tell the difference. For example, if you want to create a *derived* logo for a local Nix user group, you might be able to insert an unaltered Nix logo graphic into the local group's name in a way that does not cause confusion. But confusingly similar derived logos are not allowed. This includes entwining Nix logos with other logos, or connecting them together in a confusing manner. Logos that simply change the colours or fonts require permission from the NF Trademarks Committee.

Reviewer 2: Again, I think things like entwining the Nix logo with others is pretty obviously non-official.

- We recommend [contacting the NF](mailto:foundation@nixos.org) for permission for **all** *derived* logos to avoid placing a confusing logo into wide-spread use. Contacting us is not a requirement for the specific non-commercial uses listed above, or when using freely distributable *derived* logos that have already been approved by the NF. However, obtaining permission from the NF is required in all other uses of a *derived* logo.

# Notes

Commercial trademark uses that predate this policy (prior to June 2023) do not require permission from the NF if the use is consistent with this policy. However, if you think you may have used the NF trademarks in the past in ways that would violate this policy, we recommend seeking permission. Although we are not generally in the business of suing for past infringement of our trademarks, the NF does reserve the right to deny trademark use that violates this policy. Past use in violation of this policy does not confer a right to continue that use.

# Helping Out

> As a member of the Python community, please keep an eye out for questionable uses of the Nix logo and the “Nix” word mark. You can report potential misuse to [the NixOS foundation board](mailto:foundation@nixos.org). We will evaluate each case and take appropriate action.

Reviewer 2: s/NixOS/Nix/
